.github/workflows/stale-prs.yml

```diff
@@ -0,0 +1,16 @@
+name: 'Manage stale PRs'
+on:
+  schedule:
+    - cron: '30 1 * * *'
+
+jobs:
+  stale:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/stale@v7
+        with:
+          days-before-pr-stale: 14
+          days-before-pr-close: 14
+          stale-pr-label: 'stale-pr'
+          stale-pr-message: 'This pull request is stale and will be closed in 14 days if no action is taken. To keep it open, leave a comment or remove the "stale" label.'
```

Comment:
We might want to add instructions for the author to ping a developer if it's ready to be merged.

Reply:
How would you phrase it?

Reply:
Something like:
```suggestion
          stale-pr-message: 'This pull request is stale and will be closed in 14 days if no action is taken. To keep it open, leave a comment or remove the "stale" label. If you\'re awating feedback from a developer, please send us a reminder (either here or on Discord).'
```